app/helpers/episodes_helper.rb

```diff
@@ -0,0 +1,22 @@
+require "text_sanitizer"
+
+module EpisodesHelper
+  def episode_itunes_type_options
+    Episode::VALID_ITUNES_TYPES.map { |val| [I18n.t("helpers.label.episode.itunes_types.#{val}"), val] }
+  end
+
+  def episode_status_class(episode)
+    case episode.publishing_status_was
```

Comment:
Bit tricky ... but I needed the color of the status box to match the _previous_ publishing_status, when you submit something invalid.